rawx/handler_chunk.go

```diff
@@ -94,9 +94,13 @@ func dumpBuffer(dst io.Writer, buf []byte) (written int, err error) {
 	return written, err
 }
 
-func copyReadWriteBuffer(dst io.Writer, src io.Reader, buf []byte) (written int64, err error) {
+func copyReadWriteBuffer(dst io.Writer, src io.Reader, pool bufferPool) (written int64, err error) {
+	buf := pool.Acquire()
+	defer pool.Release(buf)
+
 	for {
 		// Fill the buffer
+		buf = buf[:cap(buf)]
```

Comment:
It's not already done with `Acquire`?

Reply:
Yes it is! This is strain, a remainder of the development process.